.github/workflows/wear_instrumentation.yml

```diff
@@ -0,0 +1,65 @@
+name: Android, Wear and JVM CI
+
+on:
+  pull_request:
+  workflow_dispatch:
+
+# Cancel any current or previous job from the same PR
+concurrency:
+  group: wear-instrumentation-${{ github.head_ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    runs-on: macos-latest
+    # TODO change to a test specific label
+    #if: contains(github.event.pull_request.labels.*.name, 'enhancement')
```

Comment:
@joreilly  can you grant me enough rights to set labels, no point running these on every build, but selectively, and maybe daily/weekly?

Reply:
And maybe make a "test" label?

Reply:
@yschimke I just added you as a collaborator to the project (had thought I'd done that before).  I also created "test" label